backend/controllers/attendance.js

```diff
@@ -2,42 +2,23 @@ const moment = require('moment')
 const { switchDateFormat, pascalToString } = require('../utils')
 const log = require('../log')
 
-const attendanceFactory = (elite2Api, whereaboutsApi) => {
-  const postAttendance = async (context, attendance) => {
-    if (!attendance || !attendance.offenderNo) {
-      throw new Error('Offender number missing')
+const attendanceFactory = whereaboutsApi => {
+  const updateAttendance = async (context, attendance) => {
+    if (!attendance || !attendance.bookingId) {
+      throw new Error('Booking ID is missing')
     }
```

Comment:
Is there a reason we're not showing an error message if an attendance is missing?